### PR TITLE
fix resets

### PR DESCRIPTION
**Pull request by rudi-cilibrasi**

### CODE REVIEW

File: src/index.mjs [hunk 1]
```diff
@@ -189,15 +216,43 @@ import {
...
-import {
-  syncConvertIdentitiesToAddresses,
-} from './sync/index.js';
-
 import {
   cloneObjectWithHexForDisplay,
 } from './util/index.js';
 
 export {
+  ADD_IDENTITY_TO_PROFILE,
```
Comment (ethanfrey) [inline src/index.mjs]: ahh... these were not exported before. i removed that?

Comment (rudi-cilibrasi) [reply src/index.mjs]: it was interaction with the autogeneration script that made src/index.mjs automatically.

**Comment by ethanfrey** [comment]
This broke something.

If I run iov-wallet against `ts-move-helpers` branch, it works
If I run it against `master` it fails.

The only changes on master since `ts-move-helpers` was this PR.

And I spent the last 3 hours trying to fix my PR built on top of this...

**Comment by rudi-cilibrasi** [comment]
i don't mind if you revert mine.

The bugs it tried to fix were also "real" in the same sense of the facade init waiting bug before so probably needs fixing in other parts also.

**Comment by rudi-cilibrasi** [comment]
BTW i did test my changes in blockly and they definitely performed as expected with the layout
performFullBoot, performCloseAndResetReduxState, performResetReduxState, performCloseResetAndReboot

**Comment by ethanfrey** [comment]
Logic does seem correct, better.

Iov wallet broke with missing symbols after this pr, not broken awaits.

**Pull request merged**